### PR TITLE
Fix types release

### PR DESCRIPTION
**Pull request by Dashron**
[![PR App][icn]][demo] | Fix RM-XYZ
:-------------------:|:----------:

## 🧰 Changes

fix the release bugs.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "DOM.iterable"
     ],
     "module": "es2020",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```
Comment (Dashron) [inline tsconfig.json:15]: https://github.com/rollup/rollup/issues/5199#issuecomment-1782231154 🤷

**Review by kellyjosephprice — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (erunion) [reply tsconfig.json:15]: we use `bundler` in all of our oas tooling as well fwiw

Comment (Dashron) [reply tsconfig.json:15]: that makes me feel a lot better about this, thank you!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.26</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.26)
<br>

**Pull request merged**